dashboard/cron.py

```diff
@@ -110,7 +88,7 @@ def update_with_udw_user(self):
         status += deleteAllRecordInTable("user")
 
         # loop through multiple course ids
-        for UDW_course_id in settings.DEFAULT_UDW_COURSE_IDS:
+        for UDW_course_id in db_util.get_supported_courses():
```

Comment:
I do not see the get_supported_courses() function defined in db_util.

Reply:
I don't either, I think it was missing from some other change on #305, I'll need to merge that first, this was based on that commit.

Reply:
I'll merge that and rebase this and it should be there.